UI/GameSettingsScreen.cpp

```diff
@@ -287,6 +287,11 @@ void GameSettingsScreen::CreateViews() {
 	CheckBox *depthRange = graphicsSettings->Add(new CheckBox(&g_Config.bDepthRangeHack, gs->T("Depth Range Hack (Phantasy Star Portable 2)")));
 	depthRange->SetDisabledPtr(&g_Config.bSoftwareRendering);
 
+	static const char *bloomHackOptions[] = { "Off", "Safe", "Balanced", "Aggressive" };
+	PopupMultiChoice *bloomHack = graphicsSettings->Add(new PopupMultiChoice(&g_Config.iBloomHack, gs->T("Bloom Effect Smoothing"), bloomHackOptions, 0, ARRAY_SIZE(bloomHackOptions), gs, screenManager()));
```

Comment:
I had no idea what bloom was until someone explained it to me in an issue here.  And I'm a generally knowledgeable doofus.

Maybe "Lower resolution for effects (reduces artifacts)"?

-[Unknown]